test/ledger/grants.integration.test.js

```diff
@@ -0,0 +1,109 @@
+import { serial as test } from 'ava'
+import BigNumber from 'bignumber.js'
+import crypto from 'crypto'
+import anonize from 'node-anonize2-relic'
+import { v4 } from 'uuid'
+import { sign } from 'http-request-signature'
+import tweetnacl from 'tweetnacl'
+import {
+  ledgerAgent,
+  uint8tohex,
+  timeout,
+  ok
+} from '../utils'
+const promotionId = 'c96c39c8-77dd-4b2d-a8df-2ecf824bc9e9'
+test('ledger: grants: add expired grant', async t => {
+  t.plan(0)
+  const url = '/v1/grants'
+  // expired grant
+  const expired = {'grants': ['eyJhbGciOiJFZERTQSIsImtpZCI6IiJ9.eyJhbHRjdXJyZW5jeSI6IkJBVCIsImdyYW50SWQiOiI0Y2ZjMzFmYy1mYjE1LTRmMTUtOTc0Zi0zNzJiMmI0YzBkYjYiLCJwcm9iaSI6IjMwMDAwMDAwMDAwMDAwMDAwMDAwIiwicHJvbW90aW9uSWQiOiJjOTZjMzljOC03N2RkLTRiMmQtYThkZi0yZWNmODI0YmM5ZTkiLCJtYXR1cml0eVRpbWUiOjE1MjY5NDE0MDAsImV4cGlyeVRpbWUiOjE1MjUxNzYwMDB9.iZBTNb9zilKubYYwYuc9MIUHZq0iv-7DsmnNu0GakeiEjcNqgbgbg-Wc2dowlMmMyjRbXjDUIC8rK4FiIqH8CQ'], 'promotions': [{promotionId, 'priority': 0, 'active': true, 'minimumReconcileTimestamp': 1526941400000}]}
+  await ledgerAgent.post(url).send(expired).expect(ok)
+})
+
+test('ledger: grants: get known', async t => {
```

Comment:
I would rename this (when combined with above) to something having to do with test expired